src/lib/UtapiClient.js

```diff
@@ -339,7 +341,8 @@ export default class UtapiClient {
             }
             // number of objects counter
             const actionErr = results[0][0];
-            const actionCounter = results[0][1];
+            // If < 0, record numberOfObjects as though bucket were empty.
```

Comment:
If UtapiReplay does not update previously unsuccessful metrics when a 15 minute interval occurs, there is the possibility of the metrics showing a negative value for that interval. Of course, it doesn't ever make sense for `storageUtilized` or `numberOfObjects` to be recorded as a negative value.

This addresses that possibility by instead recording a value of `0` for that time period, essentially allowing some flexibility. The design is such that eventually these metrics will be updated appropriately.